test/e2e/catalog_e2e_test.go

```diff
@@ -1099,7 +1105,7 @@ var _ = Describe("Catalog represents a store of bundles which OLM can use to ins
 			}
 
 			return false, nil
-		}, 5*time.Minute, 1*time.Second).Should(BeTrue())
+		}).Should(BeTrue())
 
 		// source should be the latest we got from the eventually block
 		Expect(source.Status.Conditions).ToNot(BeNil())
```

Comment:
nit: I suspect we could refactor most of the rest of this test into a matcher for the previous `Eventually` statement.